pkg/test/framework/components/echo/call.go

```diff
@@ -78,6 +78,9 @@ type CallOptions struct {
 	// Validator for server responses. If no validator is provided, only the number of responses received
 	// will be verified.
 	Validator Validator
+
+	// CallInstance indicates whether call app directly or not.
```

Comment:
nit: This is a bit misleading name, as it only changes the port to use the instance port, it doesn't automatically make it call the workload IP?